core/ppu.cpp

```diff
@@ -132,14 +126,16 @@ void PPU::step(u8 cpuCyclesElapsed) {
         // check current scanline >= 144, then enter VBLANK, else enter OAM to prepare to draw another line
         if (scanline >= 144) {
           mode = VBLANK;
+          //Not sure if we request this here or at the start of VBLANK's `case` statement
+          cpu.requestInterrupt(Interrupt::VBLANK); 
```

Comment:
Now that I am thinking about it, I think it probably makes most sense here because once the GPU is in VBLANK I would think we would want to signal that immediately, whereas if we had it in the case statement it would have to wait until there are enough cycles to run VBLANK, but I could be wrong on my understanding of this. Either way, it probably is fine for the scope of our project.